monailabel/scribbles/utils.py

```diff
@@ -109,22 +109,46 @@ def make_iseg_unary(
     return unary_term
 
 
-def make_histograms(image, scrib, scribbles_bg_label, scribbles_fg_label, bins=32):
+def make_histograms(image, scrib, scribbles_bg_label, scribbles_fg_label, alpha=1, bins=32):
```

Comment:
It would make sense to offer the option to specify `alpha_bg` and `alpha_fg` independently. This is helpful if we have informative prior knowledge of these.

Reply:
updated to address this